### DISCUSSION TITLE
[Office Hours]

### DISCUSSION THREAD

**Discussion by ckunchur** [CS342/.github#27] [Office Hours] [answered]
### Related Discussion

Hoping to get some some help debugging navigation logic between a landing and onboarding screen! 

### Reproduction

I want to be able to navigate from a landing screen that is initially shown when the app is launched to the first onboarding screen. I initially tried binding navigation to a button and adding to StorageKeys to check if the button was pressed or not but the new key could not be recognized. 

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment, marked as answer]
Solved this in the office hours by modifying the launch sequence as shown in the https://github.com/CS342/2023-PAWS/pull/4 PR.
